components/dropdown.js

```diff
@@ -3826,26 +3847,48 @@ $.fn.dropdown.settings = {
 
 /* Templates */
 $.fn.dropdown.settings.templates = {
+  escape: function(string) {
+    var
+        badChars     = /[&<>"'`]/g,
+        shouldEscape = /[&<>"'`]/,
+        escape       = {
+          "&": "&amp;",
+          "<": "&lt;",
+          ">": "&gt;",
+          '"': "&quot;",
+          "'": "&#x27;",
+          "`": "&#x60;"
+        },
+        escapedChar  = function(chr) {
+          return escape[chr];
+        }
+    ;
+    if(shouldEscape.test(string)) {
+      return string.replace(badChars, escapedChar);
+    }
+    return string;
+  },
 
   // generates dropdown from select values
   dropdown: function(select) {
     var
       placeholder = select.placeholder || false,
-      values      = select.values || {},
-      html        = ''
+      values      = select.values || [],
```

Comment:
Why did the default value change from an object to an array?

Reply:
It looks like a bug fix, previously `var values` was set but never used because it looped directly over `select.values` instead: https://github.com/stqry/Semantic-UI-CSS/pull/1/files#diff-449ac400a1bf6f602a60c591e25d914d36f949eecfc9fd282cb19a3e803e91cfL3845.

This change is taken from https://github.com/fomantic/Fomantic-UI/pull/298/files#diff-5ee8757d7a7e983335e14411dcc4f6dda835d6132a1e7f12ce38b2a6c7215493R4031. I assume changing the default was just a minor refactor, either way the result should be the same because it's only used in the `$.each`, and it is empty so nothing happens.